components/brave_rewards/resources/shared/components/newtab/rewards_card.tsx

```diff
@@ -145,7 +145,7 @@ export function RewardsCard (props: Props) {
   }
 
   function renderBalance () {
-    if (props.grantInfo) {
+    if (props.grantInfo && props.grantInfo.amount > 0) {
```

Comment:
This will also take care of the case where `grantInfo.amount` is `undefined`, since `undefined > 0 === false`.